packages/core/package.json

```diff
@@ -26,19 +24,16 @@
   "dependencies": {
     "@loopback/context": "^4.0.0-alpha.29",
     "lodash": "^4.17.4",
-    "topo": "^3.0.0",
-    "util.promisify": "^1.0.0"
+    "topo": "^3.0.0"
   },
   "devDependencies": {
     "@loopback/build": "^4.0.0-alpha.12",
     "@loopback/testlab": "^4.0.0-alpha.22"
   },
   "files": [
     "README.md",
-    "index.js",
-    "index.d.ts",
+    "dist/index.js",
```

Comment:
Please note we have the following files under `dist`. We should include all of them except `test`. The top level `index.*` files are useful for source map.
```
index.d.ts
index.js
index.js.map
src
test
```